test/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+find_package(Qt5 COMPONENTS Core Gui Widgets REQUIRED)
+find_package(Catch2 REQUIRED)
+
+add_executable(test test_aopacket.cpp test_caseloading.cpp test_apng.cpp test_bass.cpp ../include/aopacket.h ../src/aopacket.cpp)
```

Comment:
I know you want to make the test build go super fast, but is manually adding the dependent source files really best practice? I would think that adding the rest of the AO sources with a CMake include would only add fifteen seconds of build time.

Reply:
I actually have no idea regarding what's best practice. There is a second Github workflow which creates a test build although that should take care of that: https://github.com/AttorneyOnline/AO2-Client/blob/6f5e933c18e0dcd9d5ee7f1ebb51adb70e868a09/.github/workflows/build.yml